msal4j-sdk/src/integrationtest/java/com.microsoft.aad.msal4j/TestConstants.java

```diff
@@ -38,8 +38,8 @@ public class TestConstants {
     public final static String ARLINGTON_GRAPH_DEFAULT_SCOPE = "https://graph.microsoft.us/.default";
 
 
-    public final static String B2C_AUTHORITY = "https://msidlabb2c.b2clogin.com/tfp/msidlabb2c.onmicrosoft.com/";
-    public final static String B2C_AUTHORITY_URL = "https://msidlabb2c.b2clogin.com/msidlabb2c.onmicrosoft.com/";
+    public final static String B2C_AUTHORITY = "https://msidlabb2c.b2clogin.com/msidlabb2c.onmicrosoft.com/";
+    public final static String B2C_AUTHORITY_LEGACY_FORMAT = "https://msidlabb2c.b2clogin.com/tfp/msidlabb2c.onmicrosoft.com/";
```

Comment:
Did legacy urls have ".b2clogin.com" in them? If not, we should modify this value so the execution goes to the place where tfp is checked.

Reply:
It seems like older B2C URLs could have it, since this constant has been unchanged since the original commit for it back in 2019: https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/46c0121e8dbf2609d346565c343385f23c1c3972/src/integrationtest/java/com.microsoft.aad.msal4j/TestConstants.java#L34

I think they just weren't guaranteed to have it, which is why the original B2C_MICROSOFTLOGIN_AUTHORITY used 'login.microsoftonline.com': https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/46c0121e8dbf2609d346565c343385f23c1c3972/src/integrationtest/java/com.microsoft.aad.msal4j/TestConstants.java#L41

Unfortunately it seems like ID labs updated some URLs so the old B2C_MICROSOFTLOGIN_AUTHORITY  doesn't work anymore (you fixed that when you were updating the regional formats in https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/574), so I don't think there's a way for us to test a non-'b2clogin.com' URL.